### DISCUSSION TITLE
can the server manage the multiple connections from different clients to one OPC UA server

### DISCUSSION THREAD

**Discussion by Ronggiao** [frangoteam/FUXA#1544] [Q&A]
for example, 8 clients visit the same OPC UA server created by the PLC, how many connections will the web server create? 8 connections for each client? or just one connection, the web server connect to the PLC, and query, subscribe or input the data based the clients request.  For details, if one client subscribe a variable named A, another client also subscribe the same variable, the web server will subscribe once for the variable A?  if the OPC UA notify the web server about the new value, the web server then tell the subscribed clients

**Comment by unocelli** [comment]
Hi, the server makes one connection, subscribes to all registered tags and sends the values of the tags to the clients according to the request (Tags displayed in View)
